ui/tests/integration/components/auth-jwt-test.js

```diff
@@ -127,35 +122,46 @@ module('Integration | Component | auth jwt', function (hooks) {
   });
 
   test('oidc: test role: it renders', async function (assert) {
```

Comment:
I wrapped these tests in a `for...` loop which ran 10+ times. I was able to get this to flake locally so since it survived the for loop I'm fairly confident this will resolve the flakiness on main